javascript/modules/client/test/integration/client-addressList.test.ts

```diff
@@ -76,32 +77,29 @@ const grapqhlEndpoints = {
 const TEST_WALLET =
   "0xdf57089febbacf7ba0bc227dafbffa9fc08a93fdc68e1e42411a14efcf23656e";
 
-const contextParams: ContextPluginParams = {
+const contextParams: ContextParams = {
```

Comment:
Did we also update the readme?

Reply:
No, but the readme was already correct :man_facepalming: